packages/react-query/src/useSuspenseInfiniteQuery.ts

```diff
@@ -30,15 +31,13 @@ export type UseSuspenseInfiniteQueryOptions<
   TError = unknown,
   TData = TQueryFnData,
   TQueryKey extends QueryKey = QueryKey
-> = Omit<
-  UseInfiniteQueryOptions<TQueryFnData, TError, TData, TQueryFnData, TQueryKey>,
-  'suspense' | 'queryKey' | 'queryFn'
->
+> = Omit<UseInfiniteQueryOptions<TQueryFnData, TError, TData, TQueryFnData, TQueryKey>, 'suspense'>
```

Comment:
From now on, UseSuspenseInfiniteQueryOptions will also support queryKey and queryFn.